datafusion/core/tests/window_fuzz.rs

```diff
@@ -329,9 +329,10 @@ async fn run_window_test(
     }
 
     let concat_input_record = concat_batches(&schema, &input1).unwrap();
-    let exec1 = Arc::new(
-        MemoryExec::try_new(&[vec![concat_input_record]], schema.clone(), None).unwrap(),
-    );
+    let memory_exec =
```

Comment:
With new changes `MemoryExec` now supports `output_ordering`. Since in these tests input is sorted. I give this information to `MemoryExec`.